tests/test_token_integration.py

```
@@ -0,0 +1,524 @@
+                )
+            ]
+
+            # Ensure no environment variables interfere
+            env_vars_to_clear = ['GH_TOKEN', 'GITHUB_TOKEN', 'GH_ACCESS_TOKEN']
+            with patch.dict(os.environ, {var: "" for var in env_vars_to_clear}, clear=False):
+                # Create manager without explicit token (should fallback to gh CLI)
+                manager = TokenManager()
+
+                # Should successfully get token from gh CLI
+                token = manager.get_token()
+                assert token == "ghp_FAKE_CLI_FALLBACK_TOKEN"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "token"

(S105)